stable/nginx-ingress/values.yaml

```diff
@@ -163,6 +163,8 @@ controller:
 
     loadBalancerIP: ""
     loadBalancerSourceRanges: []
+    loadBalancerDisableHTTP: false
+    loadBalancerDisableHTTPS: false
```

Comment:
It's not really related to load balancer settings. I'd also go the other way around and do `enable` instead of `not disable`. I'd do this:

```yaml
enableHttp: true
enableHttps: true
```